inst/templates/ad_adcm.R

```diff
@@ -75,7 +75,7 @@ adcm <- cm %>%
     trunc_out = FALSE
   )
 
-# Derive flags
+## Derive flags ----
```

Comment:
i would delete line 35 - its a repeat of this and its for a section about dates - that would match adae template then

Reply:
Deleted, good catch

Reply:
@bundfussr did you commit this change? i still see it. also ADPP update still needed (see comment above) and there's a URL check failing, but guess you're working through these

Reply:
Yes, the change and the update of ADPP is in the https://github.com/pharmaverse/admiral/pull/1343/commits/696e42e158ede6984724bd986435fd17ad9a34b3 commit.

Regarding the URL check I am not sure why it is failing. I did not change any of the URLs.

Reply:
I think cran/microsoft changed the urls for the snapshots.  
![image](https://user-images.githubusercontent.com/10111024/184163274-66a28cda-10f5-44cf-8a60-9e893183c796.png)

Reply:
Do you know the new ones?

Reply:
I think the snapshot/time machine part of this website isn't available right now.  Maybe just an issue on their end.
![image](https://user-images.githubusercontent.com/10111024/184164808-5f1852da-54db-45c7-897a-05cf0c67535c.png)